modules/dists/BlockCycDist.chpl

```diff
@@ -1102,100 +1110,156 @@ class LocBlockCyclicArr {
   const numblocks : rank*int = allocDom._lens;
 
   const sizes : (rank+1)*int = allocDom._sizes;
+  const localeIndex: if rank == 1 then int else rank*int;
```

Comment:
I am a bit surprised this compiles. I would expect `type localeIndex`

Reply:
It's declaring a variable, not a type; it's just doing so while specifying the type.

Reply:
Ugh... sorry I got confused